package.json

```diff
@@ -18,8 +18,9 @@
     "build:create-tar-file-from-deps": "tar -czf ./deps.tar.gz ./deps && mv ./deps.tar.gz ./deps",
     "build": "npm run build:js && npm run build:backends && npm run build:create-tar-file-from-deps",
     "build:activated": "npm run build:js && npm run build:backends-activated",
+    "postbuild:add-exec-permission": "chmod +x ./postbuild_fixups.sh",
     "postbuild:fixups": "./postbuild_fixups.sh $npm_package_version",
-    "postbuild": "npm run postbuild:fixups"
+    "postbuild": "npm run postbuild:add-exec-permission && npm run postbuild:fixups"
```

Comment:
Hmm, why didn't you want to `git add` the file after `chmod +x` instead?

Reply:
mmh.. Because it was already `+x` locally. I assumed the issue came from the build docker container cloning the repo with the file without the right execution permission. 

I'll fix that